tests/unit/amo/components/TestSearchResult.js

```diff
@@ -149,6 +149,10 @@ describe(__filename, () => {
   it('displays a message if the theme preview image is unavailable', () => {
     const addon = createInternalAddon({
       ...fakeAddon,
+      previews: [],
+      theme_data: {
+        previewURL: null,
+      },
```

Comment:
So why exactly do we need to change that?

Reply:
Also, should we have tests withe one `preview` entry for example?

Reply:
ok so i added a test case for the alternate of this - is there is a preview - for each type